packages/web-react/src/components/Grid/demo/GridSpan.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Grid from '../Grid';
+import GridSpan from '../GridSpan';
+
+const GridSpanDemo = () => (
+  <Grid>
+    <GridSpan over={2}>
+      <div className="docs-Box docs-Box--small">2</div>
```

Comment:
Please, create the `DocsBox` component the same way the `DocsSection` and use it. This component is also prepared in web-twig (https://github.com/lmc-eu/spirit-design-system/pull/1036/files#diff-c8ce68aa31e6a644232cb5f6a6763ce6b324336fd94e116373e4714638106c5a) so it makes sense to be also here.

Reply:
Rebased and switched for `<DocsBox />`